lib/dor/text_extraction/speech_to_text.rb

```diff
@@ -52,8 +52,42 @@ def cleanup_s3_folder
       # return a list of filenames that should be stt'd
       # iterate over all files in cocina_object.structural.contains, looking at mimetypes
       # return a list of filenames that are correct mimetype
+      # then filter out any files that either (1) do not have an audio track or (2) have audio that is mostly silent
       def filenames_to_stt
-        stt_files.map(&:filename)
+        available_files = stt_files.map(&:filename)
+        available_files.select { |filename| has_audio_track?(filename) && has_useful_audio_track?(filename) }
+      end
```

Comment:
this is what filters the files to send to the speech to text to only those with an audio track that is not mostly silent